scipy/stats/_unuran/unuran_wrapper.pyi

```diff
@@ -125,3 +125,18 @@ class DiscreteAliasUrn(Method):
                  domain: None | Tuple[float, float] = ...,
                  urn_factor: float = ...,
                  random_state: SeedType = ...) -> None: ...
+
+
+class DGTDist(Protocol):
+    @property
+    def pmf(self) -> Callable[..., float]: ...
+    @property
+    def support(self) -> Tuple[float, float]: ...
+
+class DiscreteGuideTable(Method):
```

Comment:
we need to add type hints for the `ppf` method.

Reply:
Done. But I'm not 100% sure if it's done correctly, would you please double check?